src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -173,6 +173,22 @@ public Node visit(ImportDeclaration decl, Void p) {
     }
 
     // Check to see if import is used in a separate method used by the target method(s)
+    if (isUsedMethodParameterType(classFullName)) {
+      return super.visit(decl, p);
+    }
+
+    decl.remove();
+    return decl;
+  }
+
+  /**
+   * Helper method to check if the given fully-qualified class name is used as a parameter by any of
```

Comment:
I think there're missing words here. It should be "Helper method to check if the given fully-qualified class name is used as _**the type**_ of a parameter by any of..."